src/utils/dates.js

```diff
@@ -28,13 +28,15 @@ export function withinTwentyFourHours(dateLastPurchased) {
  * @param {Date} newerPurchase a JavaScript Date
  * @param {Date} olderPurchase a JavaScript Date
  * @returns {number} the whole number of days between the two input dates
+ * NOTE: will be positive if dateOne is later than dateTwo, otherwise will be negative
  */
-export function getDaysBetweenDates(newerPurchase, olderPurchase) {
-	const newerPurchaseMilliseconds = newerPurchase.getTime();
-	const olderPurchaseMilliseconds = olderPurchase.getTime();
+export function getDaysBetweenDates(dateOne, dateTwo) {
+	const dateOneMilliseconds = dateOne.getTime();
+	const dateTwoMilliseconds = dateTwo.getTime();
 
-	const timeBetween = newerPurchaseMilliseconds - olderPurchaseMilliseconds;
-	const daysBetween = Math.floor(timeBetween / ONE_DAY_IN_MILLISECONDS);
+	const timeBetween = dateOneMilliseconds - dateTwoMilliseconds;
+	const daysBetween = timeBetween / ONE_DAY_IN_MILLISECONDS;
 
-	return daysBetween;
+	// positive and negative numbers should both drop their decimals (only whole number answers)
+	return daysBetween >= 0 ? Math.floor(daysBetween) : Math.ceil(daysBetween);
```

Comment:
Just curious, do you think `Math.round` could work better here so you wouldn't need to do this calculation?

Reply:
The mathematician in me wants everything to round consistently, but I agree that `Math.round` would probably be simpler for the sake of code readability. Looking at the [MDN page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round) for `Math.round`, it still rounds slightly differently between positive and negative numbers, but it is much more consistent than using `Math.floor` for both would be. The partial day is probably not a huge deal either way for what our app is doing, so I'm open to swapping it to `Math.round` if you think that would be better.